docs/architecture.rst

```diff
@@ -122,8 +122,8 @@ Modin DataFrame API
 
 * import/export functions
    * ``from_pandas``: Convert a pandas dataframe to a Modin dataframe.
-   * ``to_pandas``: Convert a Modin dataframe to a pandas dataframe.
-   * ``to_numpy``: Convert a Modin dataframe to a numpy array.
+   * ``to_pandas``: Convert a Modin dataframe to a Pandas dataframe.
```

Comment:
I think the lowercase `pandas` is what is used by the developers: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html#pandas.DataFrame. That was the motivation for that spelling.

Reply:
Ok, agree